packages/plugin-utils/src/resolveOptions.ts

```diff
@@ -99,6 +99,10 @@ export async function resolveOptions(
       runOnStartup: true,
       transformers: [],
       extractors: [],
+      extraTransformTargets: {
+        css: [],
+        groups: [],
```

Comment:
```suggestion
        detect: [],
```

Reply:
Thanks han, just fixed that commit in a separate commit